app/assets/javascripts/sidebar.js

```diff
@@ -1,96 +1,42 @@
-/**
- * The functions here are global because they need to be
- * accesed from outside (in reports view).
- */
-
-/* global PerfectSb */
+/* global PerfectScrollbar */
 
 var Sidebar = (function() {
-  const SIDEBAR_ID = '#slide-panel';
-  const STORAGE_TREE_KEY = 'scinote-sidebar-tree-state';
-  const STORAGE_SCROLL_TREE_KEY = 'scinote-sidebar-tree-scroll-state';
+  const SIDEBAR_CONTAINER = '.sidebar-container';
 
   function reloadSidebar() {
-    let url = $('#wrapper').find('.sidebar-root').data('sidebar-url');
+    let url = $(SIDEBAR_CONTAINER).data('sidebar-url');
     $.get(url, function(result) {
-      $('#sidebar-wrapper').html(result.html);
-    });
-  }
-
-  function toggleTree($treeChildren) {
-    $treeChildren.toggleClass('hidden');
-    $.each($treeChildren, (i, treeChild) => {
-      $(treeChild).closest('.branch').find('.tree-toggle').first()
-        .toggleClass('fa-caret-down')
-        .toggleClass('fa-caret-right');
+      $(SIDEBAR_CONTAINER).find('.sidebar-body').html(result.html);
+      showSelectedLeaf();
     });
   }
 
-  function saveCurrentState() {
-    var config = [];
-    $.each($(SIDEBAR_ID).find('.tree-child:not(:hidden)'), (i, branch) => {
-      config.push(`[data-branch-id=${branch.dataset.branchId}]`);
-    });
-    sessionStorage.setItem(STORAGE_TREE_KEY, config.join(','));
-  }
-
-  function loadLastState() {
```

Comment:
You removed `loadLastState` fuction. But it is still used in `projects/index.js`. Probably the whole ajax call for refreshing sidebar tree navigation should be removed from `refreshCurrentView` function.